pyrobosim/examples/demo_astar_grid.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+import os
+
+from pyrobosim.core.yaml import WorldYamlLoader
+from pyrobosim.utils.general import get_data_folder
+from pyrobosim.utils.pose import Pose
+from pyrobosim.navigation.astar_grid import AStarGridPlanner
+
+# Load a test world.
+data_folder = get_data_folder()
+loader = WorldYamlLoader()
+world = loader.from_yaml(os.path.join(data_folder, "test_world.yaml"))
+
+
+def test_astar_grid(resolution=0.01):
```

Comment:
Don't have the default resolution this small, as it took 7 seconds on my machine. 0.05 should be good.